CHANGELOG.md

```diff
@@ -26,6 +26,12 @@
 <!--          addition of notable features. When PATCH is 0 it can      -->
 <!--          be omitted                                                -->
 
+# 13 April 2021
```

Comment:
We're actually going to release this alongside ATT&CK v9, so you can remove the release date here. It'll just live on develop until then.